backend/de.metas.contracts/src/main/java/de/metas/contracts/impl/FlatrateBL.java

```diff
@@ -1182,7 +1180,10 @@ public void extendContractAndNotifyUser(final @NonNull ContractExtendingRequest
 	{
 		if (!isExtendableContract(request.getContract()))
 		{
-			throw new AdempiereException(MSG_FLATRATE_CONDITIONS_EXTENSION_NOT_ALLOWED, request.getContract());
+			// avoid ad_issue if extension is simply not allowed in scheduled process
+			// throw new AdempiereException(MSG_FLATRATE_CONDITIONS_EXTENSION_NOT_ALLOWED, request.getContract());
+			logger.info("Skipping contract No" + request.getContract().getDocumentNo() + ", because of extension not allowed transition");
```

Comment:
1. pls use `{}` and avoid string concatenation.
2. shall we drop the MSG_FLATRATE_CONDITIONS_EXTENSION_NOT_ALLOWED AD_Message?
3. why log info and not debug?